docs/Advanced-Configuration.md

```diff
@@ -50,21 +48,9 @@ The following are required to configure OAP to use PMem cache.
   mount -o dax /dev/pmem1 /mnt/pmem1
 ```
 
-   In this case file systems are generated for 2 NUMA nodes, which can be checked by "numactl --hardware". For a different number of NUMA nodes, a corresponding number of namespaces should be created to assure correct file system paths mapping to NUMA nodes.
-   
    For more information you can refer to [Quick Start Guide: Provision Intel® Optane™ DC Persistent Memory](https://software.intel.com/content/www/us/en/develop/articles/quick-start-guide-configure-intel-optane-dc-persistent-memory-on-linux.html)
 
-#### Configuration for NUMA
 
-1. Install `numactl` to bind the executor to the PMem device on the same NUMA node. 
-
-   ```yum install numactl -y ```
-
-2. We strongly recommend you use NUMA-patched Spark to achieve better performance gain for the following 3 cache strategies. Besides, currently using Community Spark occasionally has the problem of two executors being bound to the same PMem path. 
```

Comment:
@winningsix Please take a review for removing NUMA-related info, thanks!  and we still need more modifications on the pmem path config of 'persistent-memory.xml'.